src/pray.c

```diff
@@ -47,6 +47,12 @@ static const char *godvoices[] = {
     "booms out", "thunders", "rings out", "booms",
 };
 
+static const char *hgodvoices[] = {
+    "belches", "ring-a-lings out", "squeaks", "squeals", "screeches", "drones",
+    "buzzes", "pipes", "tinkles", "rumbulates", "moos", "brays", "barks",
+    "screams", "howls", "ululates", "quacks", "meows",
+};
```

Comment:
I'd argue that the voice of a god will overwrite your hallucination.
But also that I don't like getting stuff too silly.

Reply:
Well, we have hallu_gods already...